src/Canvas.ts

```diff
@@ -48,6 +48,15 @@ class Canvas {
       this.ctx.putImageData(this.lastImageData, 0, 0);
     }
   }
+
+  public preventMobileScrolling(): void {
+    const preventDefault = (event: Event) => event.preventDefault();
+
+    this.canvasElement.addEventListener('touchstart', preventDefault);
+    this.canvasElement.addEventListener('touchmove', preventDefault);
+    this.canvasElement.addEventListener('touchend', preventDefault);
+    this.canvasElement.addEventListener('touchcancel', preventDefault);
```

Comment:
Is adding an string array like `[touchstart, touchmove, touchend, touchcancel]`, then iterate on it in a loop is cleaner ?

Reply:
Not sure it would be cleaner, guess I can do that yeah